examples/userapi/05_conditional_dimension.ipynb

```diff
@@ -0,0 +1,851 @@
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "#define _POSIX_C_SOURCE 200809L\n",
+      "#include \"stdlib.h\"\n",
+      "#include \"math.h\"\n",
+      "#include \"sys/time.h\"\n",
+      "#include \"xmmintrin.h\"\n",
+      "#include \"pmmintrin.h\"\n",
+      "#include \"omp.h\"\n",
+      "\n",
+      "struct dataobj\n",
+      "{\n",
+      "  void *restrict data;\n",
+      "  int * size;\n",
+      "  int * npsize;\n",
+      "  int * dsize;\n",
+      "  int * hsize;\n",
+      "  int * hofs;\n",
+      "  int * oofs;\n",
+      "} ;\n",
+      "\n",
+      "struct profiler\n",
+      "{\n",
+      "  double section0;\n",
+      "} ;\n",
+      "\n",
+      "\n",
+      "int Kernel(struct dataobj *restrict g_vec, struct dataobj *restrict h_vec, struct profiler * timers, const int x_M, const int x_m, const int y_M, const int y_m)\n",
+      "{\n",
+      "  float (*restrict g)[g_vec->size[1]] __attribute__ ((aligned (64))) = (float (*)[g_vec->size[1]]) g_vec->data;\n",
+      "  float (*restrict h)[h_vec->size[1]] __attribute__ ((aligned (64))) = (float (*)[h_vec->size[1]]) h_vec->data;\n",
+      "  /* Flush denormal numbers to zero in hardware */\n",
+      "  _MM_SET_DENORMALS_ZERO_MODE(_MM_DENORMALS_ZERO_ON);\n",
+      "  _MM_SET_FLUSH_ZERO_MODE(_MM_FLUSH_ZERO_ON);\n",
+      "  struct timeval start_section0, end_section0;\n",
+      "  gettimeofday(&start_section0, NULL);\n",
+      "  /* Begin section0 */\n",
+      "  for (int x = x_m; x <= x_M; x += 1)\n",
+      "  {\n",
+      "    for (int y = y_m; y <= y_M; y += 1)\n",
+      "    {\n",
+      "      if (g[x + 1][y + 1] != 0 && y < 5)\n",
+      "      {\n",
+      "        h[x + 1][y + 1] = g[x + 1][y + 1] + h[x + 1][y + 1];\n",
+      "      }\n",
+      "    }\n",
+      "  }\n",
+      "  /* End section0 */\n",
+      "  gettimeofday(&end_section0, NULL);\n",
+      "  timers->section0 += (double)(end_section0.tv_sec-start_section0.tv_sec)+(double)(end_section0.tv_usec-start_section0.tv_usec)/1000000;\n",
+      "  return 0;\n",
+      "}\n",
+      "\n",
+      "[[1. 0. 0. 0. 0. 0. 0. 0. 0. 0.]\n",
+      " [0. 1. 0. 0. 0. 0. 0. 0. 0. 0.]\n",
+      " [0. 0. 1. 0. 0. 0. 0. 0. 0. 0.]\n",
+      " [0. 0. 0. 1. 0. 0. 0. 0. 0. 0.]\n",
+      " [0. 0. 0. 0. 1. 0. 0. 0. 0. 0.]\n",
+      " [0. 0. 0. 0. 0. 0. 0. 0. 0. 0.]\n",
+      " [0. 0. 0. 0. 0. 0. 0. 0. 0. 0.]\n",
+      " [0. 0. 0. 0. 0. 0. 0. 0. 0. 0.]\n",
+      " [0. 0. 0. 0. 0. 0. 0. 0. 0. 0.]\n",
+      " [0. 0. 0. 0. 0. 0. 0. 0. 0. 0.]]\n"
+     ]
+    }
+   ],
+   "source": [
+    "#NBVAL_IGNORE_OUTPUT\n",
+    "\n",
+    "h = Function(name='h', shape=grid.shape, dimensions=(x, ci))\n",
+    "\n",
+    "op = Operator(Eq(h, h + g))\n",
+    "op.apply()\n",
+    "\n",
+    "print(op.ccode)\n",
+    "print(h.data[:,:])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Use case B (Function subsampling)\n",
```

Comment:
I'd clean up the language a bit in this section and make the example a bit more informative. i.e. subsample some function used in an operator and then plot show the properties of, and plot, the subsampled field.

Reply:
I printed some data. Instead of a field I think just a small array is enough, same as the docstring.
What properties exactly you refer to?